intermatic-pe653-pool-control-system.groovy

```diff
@@ -1500,16 +1637,18 @@ private def List togglePoolSpaModeInternal() {
 }
 
 /** Light Color */
-private List setLightColorInternal(int col) {
+private List setLightColorInternal(col) {
```

Comment:
Keeping types in the signature gives us free coercion I think? Or clear errors at least if not.